source/kraken/apply_disruption.cpp

```diff
@@ -182,6 +182,21 @@ static std::string concatenate_impact_uris(const nt::MetaVehicleJourney& mvj) {
     return impacts_uris.str();
 }
 
+
+static nt::Route* get_or_create_route(const nt::disruption::Impact& impact, nt::PT_Data& pt_data) {
+    nt::Network* network = pt_data.get_or_create_network("network:additional_service", "additional service");
+    nt::CommercialMode* comm_mode = pt_data.get_or_create_commercial_mode("commercial_mode:additional_service",
+                                                                          "additional service");
+    // TODO: manage line.code when necessary
```

Comment:
This should not be necessary for SNCF data at least (empty in base_schedule data)